translations/strings.yaml

```diff
@@ -697,3 +698,6 @@ global:
     value: Go back
     comment: Label for button used in intro modals to return to the app.
 
+getHelp:
+  helpCenter: Help Center
+  viewLogs: View Logs
```

Comment:
I assume this is a button or a link? Adding it to the ID or a comment would help, because English can be confusing.